schemagen/pkg/gen/schema.go

```diff
@@ -52,15 +52,17 @@ func GenerateSchema(packageDir string) schema.PackageSpec {
 		Language: map[string]schema.RawMessage{
 			"csharp": rawMessage(map[string]interface{}{
 				"packageReferences": map[string]string{
-					"Pulumi":     "3.*",
-					"Pulumi.Aws": "5.*",
+					"Pulumi":        "3.*",
+					"Pulumi.Aws":    "5.*",
+					"Pulumi.Docker": "3.*",
 				},
 				"liftSingleValueMethodReturns": true,
 			}),
 			"go": rawMessage(map[string]interface{}{
 				"generateResourceContainerTypes": true,
 				"importBasePath":                 "github.com/pulumi/pulumi-awsx/sdk/go/awsx",
 				"liftSingleValueMethodReturns":   true,
+				"internalDependencies":           []string{"github.com/pulumi/pulumi-docker/sdk/v3/go/docker"},
```

Comment:
Is there any reason why we don't need to specify aws as well here?

Reply:
Because the SDK directly references types from the AWS provider - so it's got a natural dependency already: https://github.com/pulumi/pulumi-awsx/pull/841/files#diff-7b5028b779639efd98a7f47865b168ca39285e1c46d94140c5d38c516d00b127L8